ansible/roles/osx-kvm/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: (re)start VM
```

Comment:
I prefer logically-related and simple things to be in a single file (no jumping around in the editor and mentally).